core/View.php

```diff
@@ -364,6 +364,7 @@ protected function applyFilterCacheBuster($output)
 
         $pattern = array(
             '~<script type=[\'"]text/javascript[\'"] src=[\'"]([^\'"]+)[\'"]>~',
+            '~<script type=[\'"]text/javascript[\'"] src=[\'"]([^\'"]+)[\'"] defer>~',
```

Comment:
This order actually breaks the chunked files as it already replaces what should be replaced by the rule that follows 2 lines below. Guess you need to put this line down the list.

Reply:
Good spot @sgiehl, thanks for pointing that out. Of course we need to cater for the more specific first. Updated.